micromixin-transformer/src/main/java/org/stianloader/micromixin/transform/internal/annotation/MixinShadowAnnotation.java

```diff
@@ -107,14 +121,20 @@ private void apply(@NotNull MixinFieldStub source, @NotNull ClassNode target, @N
 
     @Override
     public void apply(@NotNull ClassNode to, @NotNull HandlerContextHelper hctx,
-            @NotNull MixinStub sourceStub, @NotNull T source, @NotNull SimpleRemapper remapper,
-            @NotNull StringBuilder sharedBuilder) {
-        // NOP
+                      @NotNull MixinStub sourceStub, @NotNull T source, @NotNull SimpleRemapper remapper,
+                      @NotNull StringBuilder sharedBuilder) {
+        if (!this.isMutable) return;
+
+        if (source instanceof MixinFieldStub) {
+            ((MixinFieldStub) source).field.access &= ~Opcodes.ACC_FINAL;
```

Comment:
`source` in this case refers to the original mixin class. This is in contrast to the target (here it is `to`) which is the node we actually want to transform